server/templates/details.html.tmpl

```diff
@@ -68,5 +68,8 @@
     <b class="font-bold">{{.Name}}</b>
     <span class="flex-none status-badge {{$s}}">{{$s | titlecase}}</span>
   </p>
-  <p class="text-dark-gray3 text-sm">{{or .Error .Description}}</p>
+  {{if (and .Description (ne .Status 0))}}
+  <p class="text-dark-gray3 text-sm">{{.Description}}</p>
```

Comment:
We would need to add `margin-top: 0.5em` to `text-dark-gray3` to make it look like the screenshot in #250, otherwise this description and the status description below will be right up against each other.

I couldn't actually find the right CSS file in this repo to be able to make this change myself. I did find [server/assets/css/main.css](https://github.com/palantir/policy-bot/blob/develop/server/assets/css/main.css) but this doesn't seem to be the right file because when I try to use `server/assets` as the static files it really doesn't work well. The only way I could get the CSS to work correctly was to extract the static files out of the Docker image.

I think I must be missing something so I'd appreciate it if you could point me towards the right file.

Reply:
Sorry about this, looks like we didn't update the development instructions for working on the UI.

Policy Bot uses [Tailwind](https://tailwindcss.com/) as a CSS framework, so you'll want to add the `mt-2` class to this `p` tag to add the additional top margin.

To test this out locally:

1. Install NodeJS and Yarn if you don't have them already
2. Run `yarn install`
3. Run `yarn run build` to generate the CSS files
4. Uncomment or add the following block to your local server configuration file so it uses the generated CSS files:
   ```yaml
   files:
     static: build/static
     templates: server/templates
   ```